src/components/atoms/Icon/Icon.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { FontAwesomeIcon, FontAwesomeIconProps } from '@fortawesome/react-fontawesome';
+import { IconProp } from '@fortawesome/fontawesome-svg-core';
+
+interface IconProps extends Omit<FontAwesomeIconProps, 'icon' | 'border'> {
```

Comment:
why do you want to Omit 'border' prop?

Reply:
Because it renders the icon with a strange border 🤨
<img width="400" alt="Screenshot 2020-05-05 at 18 06 56" src="https://user-images.githubusercontent.com/5938217/81088521-54458880-8efb-11ea-9c67-83dc0ab89038.png">
and I feel this could be mis-used by consumers of the library 💆🏻‍♂️